html/selionGrid.html

```diff
@@ -15,9 +15,9 @@ <h2 id="introduction">Introduction</h2>
         PhantomJS binary in each node without impacting existing test case execution.</li>
     <li>Force Restart Node – Forcibly restart the node without logging in to the
         physical node.</li>
-    <li>Self-Healing – After execution of 25 test cases, the grid node will be recycled
-        automatically to avoid unexpected issues. The number of test cases can be configured in the file
-        <code>selion.grid.properties</code></li>
+    <li>Self-Healing – Recycles the node after a pre-defined number of unique sessions are finished to avoid unexpected issues.
+    Defaults to 50. Configurable via <code>$SELION_HOME/config/SeLionConfig.json</code></li>
+    <li>View Node Logs – Grid provides the ability to view all the registered node log files</li>
```

Comment:
We should mention TransferServlet and "selion-hub-storage" since the SeLionConfig.json includes configuration for it.